WordPress/src/main/res/layout/stats_block_list_item.xml

```diff
@@ -102,20 +103,48 @@
         android:text="@string/unknown"
         android:textAlignment="viewStart"
         android:visibility="gone"
-        app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="@+id/text"
         app:layout_constraintStart_toStartOf="@+id/text"
         app:layout_constraintTop_toBottomOf="@+id/text"
         app:layout_constraintVertical_bias="0.0"
         app:layout_constraintVertical_chainStyle="packed"/>
 
+    <android.support.constraint.ConstraintLayout
```

Comment:
I think it would be much simpler to just use a `ProgressBar` instead and pass around integers from 0 - 100 instead.

Reply:
I really should have thought of that.  Quite embarrassed about that 🙁   Will try that.